script/lib/util.py

```diff
@@ -295,3 +296,22 @@ def make_version(major, minor, patch, pre = None):
     return major + '.' + minor + '.' + patch
   return major + "." + minor + "." + patch + '-' + pre
 
+def get_out_dir():
+  out_dir = 'Debug'
+  override = os.environ.get('ELECTRON_OUT_DIR')
+  if override is not None:
+    out_dir = override
+  return os.path.join(GN_SRC_DIR, 'out', out_dir)
+
+# NOTE: This path is not created by gn, it is used as a scratch zone by our
+#       upload scripts
+def get_dist_dir():
+  return os.path.join(get_out_dir(), 'gen', 'electron_dist')
```

Comment:
i don't think `gen` is the right place for this, `gen` is generally for generated _source_ files, i.e. things that will be compiled by a later step.

Reply:
@nornagon We can move this to whereever you want, as the comment implies it's not created, updated or read by anything other than our own scripts.  Changing this path is 🆗 

`gen` just seemed the closest match to our old `dist` directory we no longer have